### PR TITLE
Update Italian language

### PR DESCRIPTION
**Pull request by bovirus**
@benzino77 

Please check and merge. Thanks.

### CODE REVIEW

File: src/locales/it.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
   "stepFeaturesTitle": "Selezione funzionalità",
   "stepFeaturesBoardDesc": "Seleziona il tipo di scheda su cui è basato il progetto",
-  "stepFeaturesBoardGeneric": "Generic",
+  "stepFeaturesBoardGeneric": "Generica",
```
Comment (benzino77) [inline src/locales/it.json:24]: This one is not used and will probably be removed/replaced. Please read the discuassion on PR #185.

Comment (bovirus) [reply src/locales/it.json:24]: @benzino77 

Currently "Generic" is used in the GUI.

I propose you to merge current PR and when you will remove in master language I will update the Italian language 

Thanks.

**Comment by drhideg** [comment]
It's still translatable with this changes:
```diff
diff --git a/src/components/AppStepper/FeaturesStep/FeaturesStep.js b/src/components/AppStepper/FeaturesStep/FeaturesStep.js
index 808cca2..34eb654 100644
--- a/src/components/AppStepper/FeaturesStep/FeaturesStep.js
+++ b/src/components/AppStepper/FeaturesStep/FeaturesStep.js
@@ -246,7 +246,14 @@ class FeaturesStep extends Component {
                                   <div className={classes.radioContainer}>
                                     <FormControlLabel
                                       control={<Radio />}
-                                      label={description}
+                                      label={item.name.endsWith('generic')
+                                        ? (
+                                          <FormattedMessage id="stepFeaturesBoardGeneric">
+                                            {(txt) => txt}
+                                          </FormattedMessage>
+                                        )
+                                        : (description)
+                                      }
                                       labelPlacement="end"
                                       {...props}
                                     />

```

**Comment by bovirus** [comment]
@benzino77 

Please apply my PR and update in Italian language "Generica" instead "Generic".

Until you will leave "Generic" in master language I ask you to have the translation in Italian language.
Then to have always Italian language aligend astring list with English.

Because currently also with latest build "Generic" is used.

Then until will use it I prefer to ahve translated because these strinsg is available for translation.

**Pull request merged**